java/src/jmri/jmrit/sample/configurexml/SampleFunctionalClassXml.java

```diff
@@ -10,7 +10,7 @@
  * Handle XML persistance of SampleFunctionalClass objects.
  * <p>
  * Mote we have not (yet) updated xml/schema to provide an XML schema for this
```

Comment:
No need to change, but while you are at it, it might be worth to change `* Mote` to `* Note`.